tests/test_travis/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for cmd in "timeout 60 ./tests/test_travis/test_ps" "timeout 50 ./tests/test_travis/worker" "timeout 50 ./tests/test_travis/worker" "./tests/test_travis/error"; do
```

Comment:
What happens when the loss doesn't go down or one of the workers (or PS) crashes? I'm not sure travis is going to fail in that case.